aws/offsite/aurora-snapshots-fargate/Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM ubuntu:18.04
+FROM node:20
 
-RUN apt-get update && \
-    apt-get install -y \
-        nodejs \
-        npm
+WORKDIR /usr/src/app
```

Comment:
why this path in particular?

Reply:
It's a convention I've seen, but there's no incredibly consistent convention out there and I don't have a strong opinion. Dumping the files in `/root` felt weird though. This particular system is a script that makes API calls and doesn't require any filesystem write access on the part of the executing user, so this path seemed fine. I've also seen `/app` or `~/app` (executing user's home). Since the executing user only needs read access to the file, and since the executing user is root, pretty much any path will technically work.

Reply:
Makes sense. Might be worth dropping in a comment to explain to any future dev that this was an arbitrary decision which could be freely changed if we someday do find a consistent convention we want to adopt

Reply:
note added!